pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.596.1</version>
+        <version>1.642.3</version>
```

Comment:
IMHO it should be bumped to Parent POM 2.x instead. At least 2.something without upper bounds dependency checks. Would be helpful for other stories like JEP-200